e2e/js_image_oci/WORKSPACE

```diff
@@ -0,0 +1,66 @@
+workspace(name = "js_image_oci")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+###
+# Setup rules_js
+###
+local_repository(
+    name = "aspect_rules_js",
+    path = "../..",
+)
+
+load("@aspect_rules_js//js:repositories.bzl", "rules_js_dependencies")
+
+rules_js_dependencies()
+
+load("@rules_nodejs//nodejs:repositories.bzl", "nodejs_register_toolchains")
+
+nodejs_register_toolchains(
+    name = "nodejs",
+    node_version = "16.9.0",
+)
+
+load("@aspect_rules_js//npm:npm_import.bzl", "npm_translate_lock")
+
+npm_translate_lock(
+    name = "npm",
+    pnpm_lock = "//:pnpm-lock.yaml",
+    verify_node_modules_ignored = "//:.bazelignore",
+)
+
+load("@npm//:repositories.bzl", "npm_repositories")
+
+npm_repositories()
+
+###
+# Setup rules_oci
+###
+
+http_archive(
+    name = "contrib_rules_oci",
+    sha256 = "85197657c25b5769664c6413988b415a14d35b6dab1754b00d8ae60019936777",
+    strip_prefix = "rules_oci-0.2.1",
+    url = "https://github.com/bazel-contrib/rules_oci/archive/v0.2.1.tar.gz",
+)
+
+load("@contrib_rules_oci//oci:dependencies.bzl", "rules_oci_dependencies")
+
+rules_oci_dependencies()
+
+load("@contrib_rules_oci//oci:repositories.bzl", "LATEST_CRANE_VERSION", "LATEST_ZOT_VERSION", "oci_register_toolchains")
+
+oci_register_toolchains(
+    name = "oci",
+    crane_version = LATEST_CRANE_VERSION,
+    zot_version = LATEST_ZOT_VERSION,
+)
+
+## Pull base images
+load(":pull.bzl", "oci_pull")
```

Comment:
Is `oci_pull` going to end up in rules_oci still?

Reply:
I filed https://github.com/bazel-contrib/rules_oci/issues/63